docs/symptom-survey/contingency-tables.md

```diff
@@ -66,9 +66,6 @@ Note: If a survey item was introduced in the middle of an aggregation period,
 derived indicators will be included in aggregations for that period but will
 only use a partial week or month of data.
 
-CSVs for the month of January 2021 only use data from January 6-31 due to a
-[definitional change in a major vaccine item on January 6](./coding.md#new-items-2).
```

Comment:
Is this no longer true?

Reply:
That's correct.

Display logic for V3 was changed on Jan 6, mid-wave 6, to be shown only to respondents indicating that they had not been vaccinated. For the purposes of the contingency tables, we ignore responses to V3 from before the change. During the initial runs we enforced this by only including input data for Jan 6 or later, resulting in non-V3-derived indicators also only using data after Jan 6, hence the note in the docs.

The logic to ignore pre-Jan 6 V3 data is now included in the variable definitions, so I backfilled to the beginning of January so we don't have any partial weeks or months.

Reply:
Makes sense, thanks.